### ISSUE TITLE
.config reload

### ISSUE THREAD

**Issue by jaguero21** [tizonia/tizonia-openmax-il#397]
I have added my credentials for spotify and gmusic in the .config. When i try the examples for either I still get the add new user prompt. Is there a way to force reload the .config file?

**Comment by juanrubio** [comment]
Hi!, thanks for stopping by.

It's strange, but are you sure you are modifying the right file? Tizonia loads its config file every time it's started. Tizonia's config is expected in this location:

```
$HOME/.config/tizonia/tizonia.conf
```

If the file you are modifying is in any other location, then it won't work. Finally, if you are modifying the right file, just make sure that the user credentials are not commented out, like this:

```
# Spotify configuration
# -------------------------------------------------------------------------
# To avoid passing this information on the command line, uncomment
# and configure accordingly
#
spotify.user     = user               <-- removed '#' 
spotify.password = pass          <-- removed '#' 
```

**Comment by jaguero21** [comment]
Yes I edited the file under $HOME and not /etc. I only uncommented the
lines for user name, password and device id for Spotify and google music.

YouTube search and play functions work great. Thanks for your time.

**Comment by juanrubio** [comment]
Great, don't hesitate to ask for help here if you find any issues. I'm closing for now, but just re-open if you need to.

**Comment by alazyworkaholic** [comment]
I'm having the same problem as jaguero21
I just installed the tizonia snap on ubuntu 18.04 i386. Youtube works fine. gmusic works, but only if I supply the user, password, and device_id on the command line. I used inotifywait -e open ~/.config/tizonia/tizonia.conf to check, and tizonia is definitely NOT reading its config file. I tried specifying $TIZONIA_RC_FILE, putting it in /etc/tizonia/tizonia.conf, but it still won't work. Tizonia 0.15.0
